### PR TITLE
Save load

### PR DESCRIPTION
**Pull request by mennoo1996**
Now, when you load a project, the blocks are actually shown on screen instead of just added in the background

### CODE REVIEW

File: src/main/java/control/TimelineController.java [hunk 1]
```diff
@@ -267,7 +281,30 @@ public void load() {
...
+     * @param project the project that was loaded
+     */
+    private void addLoadedBlocks(ScriptingProject project) {
+        System.out.println("Adding loaded blocks");
+        for (int i = 0; i < project.getCameraTimelines().size();i++) {
+            System.out.println("Outer loop");
+            CameraTimeline timeline = project.getCameraTimelines().get(i);
+            int amountShots = timeline.getShots().size();
+            for (int j = 0; j < amountShots;j++) {
+                System.out.println(timeline.getShots().size());
```
Comment (bart-de-jonge) [inline src/main/java/control/TimelineController.java]: Please remove degub prints when makeing a Pull request. Maybe replace them with log.debug?

**Comment by bart-de-jonge** [comment]
I placed one small comment, other then that it looks fine :)

File: src/main/java/control/TimelineController.java (hunk 1 above)
Comment (mennoo1996) [inline src/main/java/control/TimelineController.java]: Sorry, forgot to remove them. I won't replace them all with log.debug, just a single log statement at the start of the method

File: src/main/java/gui/centerarea/CameraShotBlock.java [hunk 2]
```diff
@@ -23,6 +23,12 @@
     // THe grid this camershotblock belongs to
```
Comment (Markvanderuit) [inline src/main/java/gui/centerarea/CameraShotBlock.java]: The :)

Comment (mennoo1996) [inline src/main/java/gui/centerarea/CameraShotBlock.java]: Changed :)

**Pull request merged**